src/object/interrupt.c

```diff
@@ -218,8 +229,11 @@ void handleInterrupt(irq_t irq)
 #endif
         }
 #ifndef CONFIG_ARCH_RISCV
```

Comment:
Looking at the code, this seems a bit confusing. Why do we keep the `#ifndef CONFIG_ARCH_RISCV` here when the code block contains something that is clearly only for ARM GICv3?